### PR TITLE
Update counter.dart

### PR DESCRIPTION
**Pull request by michal-repo**
Fixed exports

## Status

**READY/IN DEVELOPMENT/HOLD**

## Breaking Changes

YES | NO

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: examples/flutter_counter/lib/counter/counter.dart [hunk 1]
```diff
@@ -1,2 +1,3 @@
 export 'cubit/counter_cubit.dart';
-export 'view/view.dart';
```
Comment (felangel) [inline examples/flutter_counter/lib/counter/counter.dart:2]: I believe this was correct as is. Why change this?

**Comment by felangel** [comment]
Closing this — no need to change this imo

File: examples/flutter_counter/lib/counter/counter.dart (hunk 1 above)
Comment (michal-repo) [reply examples/flutter_counter/lib/counter/counter.dart:2]: Tested this today with Flutter 3.3.4 and getting error "URI doesn't exist"
obraz

Comment (felangel) [reply examples/flutter_counter/lib/counter/counter.dart:2]: There is a view file: https://github.com/felangel/bloc/blob/master/examples/flutter_counter/lib/counter/view/view.dart

Comment (michal-repo) [reply examples/flutter_counter/lib/counter/counter.dart:2]: Take a look at Docs:
https://bloclibrary.dev/#/fluttercountertutorial?id=counter-view
it says:
Let's create **lib/counter/view/counter_view.dart**
not lib/counter/view/view.dart

Comment (felangel) [reply examples/flutter_counter/lib/counter/counter.dart:2]: Good catch! We should update the barrel section in the docs right after to include creating a barrel for view as well. Feel free to open a PR to fix the docs. The code is fine as is.

**Pull request closed without merging**